epub34/a11y-tech/index.html

```diff
@@ -1163,10 +1148,9 @@ <h4>Identify pages in audio playback</h4>
 						in the audio playback of a publication it is not only distracting, but can be confusing, as well
 						(e.g., the number could be read out in the middle of a sentence).</p>
 
-					<p>To mitigate this potential annoyance to readers, <a
-							href="https://www.w3.org/TR/epub/#dfn-epub-creator">EPUB creators</a> need to identify page
-						announcements in <a href="https://www.w3.org/TR/epub/#dfn-media-overlay-document">media overlay
-							documents</a> when they are included. Identification allows a <a
+					<p>To mitigate this potential annoyance to readers, identify page announcements in <a
+							href="https://www.w3.org/TR/epub/#dfn-media-overlay-document">media overlay documents</a>
+						when they are included. Identification allows a <a
```

Comment:
```suggestion
					<p>To mitigate this potential annoyance to readers, page announcements in <a
							href="https://www.w3.org/TR/epub/#dfn-media-overlay-document">media overlay documents</a>
						should be identified when they are included. Identification allows a <a
```

Isn't it more consistent to put this sentence in passive mode, too, just as it was done elsewhere? I've made an attempt to it, but you are the final arbiter...

Reply:
Ya, that's the balancing act. Passive voice is typically shunned for writing but then one of the problems we've always had is who is the actor. I'm fine either way.